### DISCUSSION TITLE
Start script

### DISCUSSION THREAD

**Discussion by BastiLoe** [OpenRadioss/OpenRadioss#834] [Ideas]
Hello all,

is there some interest in a start script similar to what Altair Radioss provides: Running starter and engine in one command, easen parallel execution, ....
Maybe somebody is already working on this?

**Comment by PaulAltair** [comment]
I created some simple bash scripts that I posted to this thread [https://github.com/orgs/OpenRadioss/discussions/496](https://github.com/orgs/OpenRadioss/discussions/496) , they run starter, engine and then also the vtk, csv converter then finally launch paraview all from a single launch command, you can easily edit them to strip the vtk and/or paraview bit off the end if you just want to run the job, hope these help!

**Comment by PaulAltair** [comment]
For this, since this post, we have got some nice gui tools: see latest gui tools discussions: [Gui thread](https://github.com/orgs/OpenRadioss/discussions/1231) and [Confluence page for guis](https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/pages/45252609/python+tk+guis+for+job+submission+anim-vtk+conversion+and+T01-csv+conversion)
